src/Symfony/Component/TypeInfo/Tests/Fixtures/DummyWithTypeAliases.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\TypeInfo\Tests\Fixtures;
+
+/**
+ * @phpstan-type CustomString = string
+ * @psalm-import-type CustomInt from DummyWithPhpDoc
+ * @phpstan-import-type CustomInt from DummyWithPhpDoc as AliasedCustomInt
+ */
+final class DummyWithTypeAliases
+{
+    /**
+     * @var Local
+     */
+    public mixed $localAlias;
+
+    /**
+     * @var CustomInt
+     */
+    public mixed $externalAlias;
+
+    /**
+     * @var AliasedCustomInt
+     */
+    public mixed $aliasedExternalAlias;
+}
+
+/**
+ * @phpstan-import-type Invalid from DummyWithTypeAliases
```

Comment:
do we support `@*-import-type Invalid from DummyWithTypeAliases as Bar` ?

Reply:
I updated tests to be more exhaustive (see `DummyWithTypeAliases`)